docs/design-patterns.mdx

```diff
@@ -62,7 +62,7 @@ pub contract NamedFields {
 
 ```
 
-[Example Code](https://github.com/onflow/flow-core-contracts/blob/master/contracts/LockedTokens.cdc#L583)
+[Example Code](https://github.com/onflow/flow-core-contracts/blob/master/contracts/LockedTokens.cdc#L718)
```

Comment:
Maybe even point to the latest commit specifically (you can get it by pressing `y` when viewing the file on GitHub), that way the link will keep working:

```suggestion
[Example Code](https://github.com/onflow/flow-core-contracts/blob/c1b31d5a4722aba3bc4ccaae9a7ec6e660887591/contracts/LockedTokens.cdc#L718)
```